Fixes for the DTL checkup and overruling of DTL for specific volumes

kvanhijf opened the pull request.

kvanhijf committed:
Fixes for the DTL checkup and overruling of DTL for specific volumes

khenderick commented on webapps/frontend/app/viewmodels/containers/vdisk.js:
```diff
@@ -107,28 +110,26 @@ define([
                         })
-                    } else if (self.dtlTarget().length === 0) {
-                        self.dtlTarget([self.dtlTargets()[0]]);
+                    } else if (self.dtlManual() === true || (self.dtlTargets().length > 0 && self.dtlManual() === false)) {
```
Can you check this again?

khenderick commented on webapps/frontend/app/viewmodels/site/vdisk-detail.js:
```diff
@@ -169,7 +181,6 @@ define([
                     .always(function() {
                         vd.loadConfiguration(true);
                     });
-                vd.oldConfiguration($.extend({}, vd.configuration()));
```
Are you sure you want these objects to be identical?

khenderick commented on webapps/frontend/app/views/site/vdisks.html:
```diff
@@ -65,7 +65,8 @@ <h2 class="pull-left" data-i18n="ovs:vdisks.title"></h2>
                     <div data-bind="status: {
                                         colors: {
-                                            green: ['ok_standalone', 'ok_sync'].contains(dtlStatus()),
+                                            blue: dtlStatus() == 'ok_standalone',
```
Maybe it's better to keep it green if manual standalone or if standalone and only one possible target, and orange if standalone but multiple targets exist.

kvanhijf committed:
Fixed the colors showing in GUI for the DTL
Green for ok_sync or ok_standalone if manually overwritten or ok_standalone when only 1 storagerouter available
Red for degraded
Orange otherwise

kvanhijf committed:
Forgot to update 1 commented line
